charts/homarr/values.yaml

```diff
@@ -165,13 +165,23 @@ livenessProbe:
     path: /api/health/live
     # -- The port on which the liveness check will be performed. This must be the same as the container port exposed by the application.
     port: 7575
+    # -- Initial delay before readiness probe is executed. increase this value if the pod is slow to fully start. 
+    initialDelaySeconds: 10
```

Comment:
Probe fields are misplaced. 
In Kubernetes, failureThreshold, initialDelaySeconds, periodSeconds, and timeoutSeconds must be siblings of httpGet, not inside it.